sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/key.py

```diff
@@ -16,9 +16,9 @@
     from typing import Any, FrozenSet
 
 class Key(object, metaclass=ABCMeta):
-    _supported_encryption_algorithms = frozenset([])  # type: FrozenSet[Any]
-    _supported_key_wrap_algorithms = frozenset([])  # type: FrozenSet[Any]
-    _supported_signature_algorithms = frozenset([])  # type: FrozenSet[Any]
+    _supported_encryption_algorithms: FrozenSet[Any] = frozenset([])
+    _supported_key_wrap_algorithms: FrozenSet[Any] = frozenset([])
+    _supported_signature_algorithms: FrozenSet[Any] = frozenset([])
```

Comment:
Since `FrozenSet` and `Any` are conditionally imported (see lines 9-16), these type hints need to be in quotes to prevent errors.
```suggestion
    _supported_encryption_algorithms: "FrozenSet[Any]" = frozenset([])
    _supported_key_wrap_algorithms: "FrozenSet[Any]" = frozenset([])
    _supported_signature_algorithms: "FrozenSet[Any]" = frozenset([])
```